src/Loggers/LogEmbeddings.jl

```diff
@@ -0,0 +1,58 @@
+"""
+
+"""
+function log_embeddings(logger::TBLogger, name::AbstractString, mat::AbstractArray; metadata=nothing, labels=nothing, metadata_header=nothing, step=nothing)
+    @assert ndims(mat) == 2 "Embedding matrix must be 2-D"
```

Comment:
Rather than this asserting,
could just dispatch on `AbstractMatrix` rather than `AbstractArray`